cime_config/allactive/config_pesall.xml

```diff
@@ -1431,8 +1431,8 @@
         </rootpe>
       </pes>
     </mach>
-  </grid>  
-  <grid name="a%ne30np4.pg2">
+  </grid>
+  <grid name="a%ne30np4">
```

Comment:
Either revert to `a%ne30np4.pg2` or change all comments in this block to reflect any `pg` modifier.